galleries/users_explain/text/mathtext.py

```diff
@@ -6,366 +6,364 @@
-
-============================== =================================
-Command                        Result
-============================== =================================
-``\acute a`` or ``\'a``        :mathmpl:`\acute a`
-``\bar a``                     :mathmpl:`\bar a`
-``\breve a``                   :mathmpl:`\breve a`
-``\dot a`` or ``\.a``          :mathmpl:`\dot a`
-``\ddot a`` or ``\''a``        :mathmpl:`\ddot a`
-``\dddot a``                   :mathmpl:`\dddot a`
-``\ddddot a``                  :mathmpl:`\ddddot a`
-``\grave a`` or ``\`a``        :mathmpl:`\grave a`
-``\hat a`` or ``\^a``          :mathmpl:`\hat a`
-``\tilde a`` or ``\~a``        :mathmpl:`\tilde a`
-``\vec a``                     :mathmpl:`\vec a`
-``\overline{abc}``             :mathmpl:`\overline{abc}`
-============================== =================================
-
-In addition, there are two special accents that automatically adjust to the
-width of the symbols below:
-
-============================== =================================
-Command                        Result
-============================== =================================
-``\widehat{xyz}``              :mathmpl:`\widehat{xyz}`
-``\widetilde{xyz}``            :mathmpl:`\widetilde{xyz}`
-============================== =================================
-
-Care should be taken when putting accents on lower-case i's and j's.  Note that
-in the following ``\imath`` is used to avoid the extra dot over the i::
-
-    r"$\hat i\ \ \hat \imath$"
-
-.. math::
-
-    \hat i\ \ \hat \imath
-
-Symbols
--------
-You can also use a large number of the TeX symbols, as in ``\infty``,
-``\leftarrow``, ``\sum``, ``\int``.
-
-.. math_symbol_table::
-
-If a particular symbol does not have a name (as is true of many of the more
-obscure symbols in the STIX fonts), Unicode characters can also be used::
-
-   r'$\u23ce$'
-
-Example
--------
-Here is an example illustrating many of these features in context.
-
-.. figure:: /gallery/text_labels_and_annotations/images/sphx_glr_mathtext_demo_001.png
-   :target: /gallery/text_labels_and_annotations/mathtext_demo.html
-   :align: center
 """
+
+# %%
+# Any text element can use math text.  You should use raw strings (precede the
+# quotes with an ``'r'``), and surround the math text with dollar signs ($), as
+# in TeX. For example:
+
+import matplotlib.pyplot as plt
+
+fig, ax = plt.subplots(figsize=(3, 3))
+fig.text(.2, .6, "plain text: alpha > beta")
+fig.text(.2, .4, r"mathtext: $\alpha > \beta$")
+ax.set(xticks=[], yticks=[])
+
+# %%
+# .. seealso::
+#
+#   :doc:`Mathtext example </gallery/text_labels_and_annotations/mathtext_demo>`
+#
+# You do not need to have TeX installed, because Matplotlib ships its own TeX
+# expression parser, layout engine, and fonts. The layout engine is a fairly
+# direct adaptation of the layout algorithms in Donald Knuth's TeX and
+# Matplotlib also provides a usetex option to call out to TeX to generate text;
+# see :ref:`usetex`
```

Comment:
The setup of this paragraph is that you don't need TeX installed; the `usetex` option is thus not an "and", but a "but", because it requires TeX. I also don't think the second half of this sentence it directly conjoined with the first, and it could be a separate sentence.
```suggestion
# direct adaptation of the layout algorithms in Donald Knuth's TeX.
# Matplotlib also provides a usetex option to call out to TeX to generate text;
# see :ref:`usetex`
```